src/components/ContentNav/__mocks__/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { SubItemProps } from '../SubItem';
+import { ItemProps } from '../Item';
+
+export const subMock1: SubItemProps = { name: 'sdgasdfasdf' };
+export const subMock2: SubItemProps = { name: 'helasd' };
+export const subMocks3: SubItemProps = [
```

Comment:
I hate long names as much as the next, but this should be called something like `subItemMocks3`, because `subMocks3` just isn't very useful to anyone. Even better, though. is if you can get rid of the first two `SubItem` mocks and just call this `subItemMock`.

Reply:
The two unused ones are removed and subMocks3 is renamed to subItemMock.